local/network.go

```diff
@@ -122,7 +122,45 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
-	defaultNetworkConfig.Genesis = string(genesis)
+
+	// update genesis validation start time
+	var genesisMap map[string]interface{}
+	if err = json.Unmarshal(genesis, &genesisMap); err != nil {
+		panic(err)
+	}
+	startTime := time.Now().Unix()
+	lockTime := startTime + 2836800
```

Comment:
What does this number mean? Lets make it a constant

Reply:
It is the difference between unlock schedule locktime and startime in local genesis of avalanchego. I copied it so the net result is for all the times of local genesis to keep the same relative relationship. Will add a comment and a constant.